sampleconfig/core.yaml

```diff
@@ -468,6 +468,11 @@ peer:
             # deliverService limits concurrent event listeners registered to deliver service for blocks and transaction events.
             deliverService: 2500
 
+    # Max message size in bytes GRPC server and client can receive
```

Comment:
```suggestion
    # Max message size in bytes GRPC server and client can receive.
    # Since all nodes should be consistent it is recommended to keep the default value of 100MB.
```